src/Momento.Sdk/Config/IConfiguration.cs

```diff
@@ -12,16 +12,47 @@ namespace Momento.Sdk.Config;
 /// </summary>
 public interface IConfiguration
 {
+    /// <inheritdoc cref="Momento.Sdk.Config.IConfiguration.LoggerFactory" />
     public ILoggerFactory LoggerFactory { get; }
```

Comment:
This looks like it's trying to inherit from itself. This should (I think) reference `Microsoft.Extensions.Logging.ILoggerFactory` instead, but I'm wondering if these crefs are necessary at all? It looked like `<inheritdoc />` was working fine without that attribute, and I think @malandis was suggesting that usage in his comment (please correct me if I'm wrong about that).

Reply:
Thank you for pointing it out!
I moved the summaries to the interfaces and forgot to remove `cref`. I will remove them.